api/v1/config/config.js

```diff
@@ -0,0 +1,31 @@
+require("dotenv").config();
+
+// Set NODE_ENV (environment variable) to 'development', 'test', or 'production' to switch mode.
+// By default, 'development' mode is assumed.
+
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    dialect: process.env.DB_DIALECT || "mysql",
+  },
+  test: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    dialect: process.env.DB_DIALECT || "mysql",
+  },
+  production: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    dialect: process.env.DB_DIALECT || "mysql",
+  },
+};
```

Comment:
This file should not be moved to this folder.
Reason: this file does not depend on API version (v1 or v2)

Reply:
Okay